packages/src/lib/navigation/navigation.component.scss

```
@@ -3,55 +3,162 @@
+        .sdg-navigation-header-actions {
+          align-self: flex-end;
+        }
+      }
+
+      @include layout.breakpoint-down(mobile) {
+        height: fit-content;
+        min-height: $mobileRowHeight;
+      }
+    }
+
+    &-actions {
+      @include layout.breakpoint-down(mobile) {
+        margin-bottom: 0;
+      }
+
+      ::ng-deep {
+        button {
+          margin-left: 24px;
+
+          @include layout.breakpoint-down(mobile) {
+            margin-left: 16px;
+          }
+        }
+
+        .mat-mdc-icon-button {
+          margin-left: 4px;
+        }
+      }
+    }
 
-.nav-link {
-  letter-spacing: 0;
-  font-size: 18px;
-  line-height: 28px;
+    .mdc-tab {
+      --mdc-secondary-navigation-tab-container-height: #{$desktopRowHeight};
 
-  &.active {
-    font-weight: bold;
+      min-width: fit-content;
+      padding-left: 8px;
+      padding-right: 8px;
+
+      @include layout.breakpoint-down(mobile) {
+        --mdc-secondary-navigation-tab-container-height: #{$mobileRowHeight};
+        padding-left: 4px;
+        padding-right: 4px;
+      }
+    }
+
+    ::ng-deep .mat-mdc-tab-links {
+      align-items: center;
+
+      & > * {
+        margin-left: 8px;
+        margin-right: 8px;
+
+        @include layout.breakpoint-down(mobile) {
+          margin-left: 4px;
+          margin-right: 4px;
+        }
+      }
+    }
   }
 
-  @include layout.breakpoint-down(mobile) {
-    font-size: 14px;
-    line-height: 20px;
+  &-tab-panel {
+    flex-grow: 1;
+    overflow: hidden;
+    align-self: flex-end;
+    height: inherit;
+
+    @include layout.breakpoint-down(mobile) {
+      width: 100%;
+      flex-grow: unset;
+      height: fit-content;
+      min-height: $mobileRowHeight;
+    }
   }
-}
 
-.navigation-actions {
-  margin-bottom: 4px;
+  &-tab-link {
+    letter-spacing: 0;
+    font-size: 18px;
+    line-height: 28px;
 
-  @include layout.breakpoint-down(mobile) {
-    margin-bottom: 0;
+    mat-icon {
+      $size: 18px;
+      height: $size;
+      width: $size;
+      font-size: $size;
+    }
+
+    &.active {
+      font-weight: bold;
+    }
+
+    &:hover {
+      ::ng-deep .mdc-tab-indicator__content--underline {
+        height: 3px;
+        opacity: 1;
+      }
+    }
   }
 
-  ::ng-deep {
-    button {
-      margin-left: 24px;
+  &-more-links {
+    padding: 0 8px;
+    min-width: fit-content !important;
+    background-color: colors.$blue-pale !important;
+    color: colors.$blue-piv !important;
 
-      @include layout.breakpoint-down(mobile) {
-        margin-left: 16px;
+    &-container {
+      display: flex;
+      align-items: center;
+
+      mat-icon {
+        margin-left: 4px;
       }
     }
+  }
 
-    .mat-mdc-icon-button {
-      margin-left: 4px;
+  &-tab-link,
+  &-more-links {
+    --mdc-filled-button-container-height: 40px;
+
+    @include layout.breakpoint-down(mobile) {
+      --mdc-filled-button-container-height: 24px;
+      font-size: 14px;
+      line-height: 20px;
```

```suggestion
      font-size: 13px;
      line-height: 16px;
```
Au format mobile, la taille devrait être de 13px et l'interligne à 16px;